pkg/cmd/codespace/codespace_selector.go

```diff
@@ -25,6 +26,7 @@ func AddCodespaceSelector(cmd *cobra.Command, api apiClient) *CodespaceSelector
 
 	cmd.PersistentFlags().StringVarP(&cs.codespaceName, "codespace", "c", "", "Name of the codespace")
 	cmd.PersistentFlags().StringVarP(&cs.repoName, "repo", "R", "", "Filter codespace selection by repository name (user/repo)")
+	cmd.PersistentFlags().StringVar(&cs.repoOwner, "repo-owner", "", "Filter codespace selection by repository owner (username or org)")
 
 	cmd.MarkFlagsMutuallyExclusive("codespace", "repo")
```

Comment:
Missed this originally, but we should have mutual-exclusion of `codespace` and `repo-owner` here too

Reply:
Pushed a change to add this